source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -247,8 +209,9 @@ void AuthenticatorImpl::doneWithStatus(const Status& status) {
 
 } // namespace
 
-AuthenticatorPtr Authenticator::create(FilterConfigSharedPtr config) {
-  return std::make_unique<AuthenticatorImpl>(config);
+AuthenticatorPtr Authenticator::create(FilterConfigSharedPtr config,
+                                       Common::JwksFetcher::JwksFetcherPtr& fetcher) {
+  return std::make_unique<AuthenticatorImpl>(config, fetcher);
```

Comment:
should we call std::move for fetcher

Reply:
Changed in e6b6b0 as per review comment.